interface/src/app/node/node_search.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * "adage.node_search" module.
+ */
+
+angular.module('adage.node_search', [
+  'ui.router',
+  'ui.bootstrap',
+  'ngResource',
+  'adage.mlmodel.components',
+  'adage.utils'
+])
+
+.config(['$stateProvider', function($stateProvider) {
+  $stateProvider.state('node_search', {
+    url: '/node/search',
```

Comment:
My original plan was to create an abstract state as parent and two child states for node information and node search. But I gave up for two reasons:

1) The two states "node" and "node_search" do not really share anything, so a parent state doesn't seem to be helpful. (I feel that their relationship is similar to the one between `gene_search` and `gene_network`.)

2) The file `node.js` has been pretty long now (475 lines). Adding the new `node_search` state to it will make the file longer. Keeping them separate will make it easier to maintain.

Does this make sense?